src/components/schedule/class/ClassCard.tsx

```diff
@@ -54,6 +55,24 @@ const ClassCard = ({
         onUpdateConfig(!selected);
     }
 
+    const shortenMiddleNames = (name: string): string =>
```

Comment:
This should be moved to a utils file or something

Reply:
Addressed in https://github.com/mathiazom/rezervo-web/pull/114/commits/593f3fe429586423e00da7135c86941718a8ff58. Also rebased this so we're up to date 🚀